content/docs/command-reference/run.md

```diff
@@ -6,13 +6,16 @@ command.
 ## Synopsis
 
 ```usage
-usage: dvc run [-h] [-q | -v] [-n <name>] [--no-exec] [--no-run-cache]
-               [--no-commit] [-d <path>] [-o <filename>] [-O <filename>]
-               [-p [<filename>:]<params_list>] [-m <path>] [-M <path>]
-               [--plots <path>] [--plots-no-cache <path>] [--live <path>]
-               [--live-no-cache <path>] [--live-no-html]
-               [-w <path>] [-f] [--outs-persist <filename>]
-               [--outs-persist-no-cache <filename>] [-c <filename>]
+usage: dvc run [-h] [-q | -v] [-n <name>] [-f]
+               [--no-exec] [--no-run-cache] [--no-commit]
+               [-d <path>] [-p [<filename>:]<params_list>]
+               [-o <filename>] [-O <filename>] [-c <filename>]
+               [--outs-persist <filename>]
+               [--outs-persist-no-cache <filename>]
+               [-m <path>] [-M <path>]
+               [--plots <path>] [--plots-no-cache <path>]
+               [--live <path>] [--live-no-cache <path>]
+               [--live-no-html] [-w <path>]
```

Comment:
Is the idea to put all the dependency and output flags together and in some meaningful order?

Reply:
Yep